platform/linuxbsd/os_linuxbsd.cpp

```diff
@@ -125,15 +125,26 @@ Error OS_LinuxBSD::shell_open(String p_uri) {
 	Error ok;
 	List<String> args;
 	args.push_back(p_uri);
-	ok = execute("xdg-open", args, false);
+	// Agnostic
+	ok = execute("xdg-open", args, true);
 	if (ok == OK) {
 		return OK;
 	}
-	ok = execute("gnome-open", args, false);
+	// GNOME
+	ok = execute("gio open", args, true);
```

Comment:
Does this work? Or should `open` be added to `args`?

Reply:
Good catch, it doesn't work this way indeed. I've added the `open` part to the args.

I didn't catch this because there is another issue actually; if `execute` fails *(such as, if `xdg-open` doesn't exist)*, it still returns `OK`, so the code doesn't attempt any other option. I'll fix that too.

Reply:
Well, I've found a rabbit hole; on `os_unix.cpp`, the exitcode was also returning 0, because of a "quirk" of `WEXITSTATUS`.

The fix is a one-liner, but it is unrelated to this PR; should I open up another one for it?

Reply:
Yeah a separate PR would be best.